src/renderer/components/preferences/general.js

```diff
@@ -140,7 +144,9 @@ class General extends PureComponent {
           type="number"
           min="0"
           defaultValue={DEFAULT_CLIPBOARD_CLEAR_SECONDS}
-          onChange={e => setSecondsUntilClearClipboard(e.target.value)}
+          onChange={e =>
+            setSecondsUntilClearClipboard(e.target.value, 'number')
```

Comment:
Why not parse this before sending it in, rather than adding a param to the function? :)